### PR TITLE
Warning Message

### PR DESCRIPTION
**Pull request by tt-le**
Added warning message thrown by test case and explaining that it is expected.

### CODE REVIEW

**Review by AustinSeto — APPROVED** [review]
LGTM

File: README.adoc [hunk 1]
```diff
@@ -313,6 +313,8 @@ You see the following output:
 
 [source, role="no_copy"]
 ----
+[INFO] Running it.io.openliberty.guides.health.HealthIT
+[INFO] [WARNING ] CWMH0052W: The class com.ibm.ws.microprofile.health.impl.HealthCheckResponseImpl implementing HealthCheckResponse in the guide-microprofile-health application in module guide-microprofile-health.war, reported a DOWN status with data Optional[{}].
```
Comment (gkwan-ibm) [inline README.adoc:317]: @tt-le  These 2 lines should be under `[INFO] T E S T S`

File: README.adoc [hunk 2]
```diff
@@ -321,13 +323,18 @@ You see the following output:
...
+[INFO] Could not send Message.
+[INFO] [err] The specified host is unknown.
 [INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.102 s - in it.io.openliberty.guides.inventory.InventoryEndpointIT
 [INFO]
 [INFO] Results:
 [INFO]
 [INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning messages are expected. The first results from [hotspot=testIfInventoryServiceIsDown file=0]`testIfInventoryServiceIsDown()` making a request to a service that is down due to being in maintenance. The second warning results from a request to a bad or an unknown host name.
```
Comment (gkwan-ibm) [inline README.adoc:336]: - is it better to say "The first warning results..."?
- "The second warning..." statement better mention from a test `testUnknownHost()` from `InventoryEndpointIT`

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]

File: README.adoc [hunk 3]
```diff
@@ -317,17 +318,25 @@ You see the following output:
...
+[INFO] Could not send Message.
+[INFO] [err] The specified host is unknown.
 [INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.102 s - in it.io.openliberty.guides.inventory.InventoryEndpointIT
 [INFO]
 [INFO] Results:
 [INFO]
 [INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning messages are expected. The first warning results from `testIfInventoryServiceIsDown()` making a request to a service that is down due to being in maintenance. The second warning and error results from a request to a bad or an unknown host name. This request is made in the `testUnknownHost()` test from `InventoryEndpointIT` integration test.
```
Comment (gkwan-ibm) [inline README.adoc:336]: @tt-le would you rewrite based on the feedback from  https://github.com/OpenLiberty/guide-microprofile-openapi/pull/101?

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 3 above)
Comment (tt-le) [reply README.adoc]: Done. What do you think ?

File: README.adoc [hunk 4]
```diff
@@ -317,17 +318,25 @@ You see the following output:
...
+[INFO] Could not send Message.
+[INFO] [err] The specified host is unknown.
 [INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.102 s - in it.io.openliberty.guides.inventory.InventoryEndpointIT
 [INFO]
 [INFO] Results:
 [INFO]
 [INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning messages are expected. The first warning results from a request to a service that is under maintenance. This request is made in the `testIfInventoryServiceIsDown()` test from `InventoryEndpointIT` integration test. The second warning and error results from a request to a bad or an unknown host name. This request is made in the `testUnknownHost()` test from `InventoryEndpointIT` integration test.
```
Comment (Charlotte-Holt) [inline README.adoc:336]: Update

>This request is made in the `testIfInventoryServiceIsDown()` test from `InventoryEndpointIT` integration test. The second warning and error results from a request to a bad or an unknown host name. This request is made in the `testUnknownHost()` test from `InventoryEndpointIT` integration test.

to

>This request is made in the `testIfInventoryServiceIsDown()` test from the `InventoryEndpointIT` integration test. The second warning and error results from a request to a bad or an unknown hostname. This request is made in the `testUnknownHost()` test from the `InventoryEndpointIT` integration test.

**Review by Charlotte-Holt — COMMENTED** [review]
@tt-le I added ID review feedback. Let me know if you have any questions. Thanks!

File: README.adoc (hunk 4 above)
Comment (gkwan-ibm) [reply README.adoc]: updated

**Review by Charlotte-Holt — APPROVED** [review]

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**